_includes/overall-schedule.md

```diff
@@ -23,5 +23,38 @@ Organizers will inform participants of the week schedule by email.
 {%- capture date %}{% if c.type != "Mentor-Mentee" %}{{ c.date }} {% if c.time %}([{{ c.time }} European Time](https://arewemeetingyet.com/Berlin/{{ c.date | date: "%Y-%m-%d" }}/{{ c.time }}/{{ cohort }}%20{{ c.type }}%20Call%20(Week%20{{ w[0] }}))){% endif %}{% endif %}{% endcapture %}
 | {{ w-desc }} | [{{ c.type }}](/{{ cohort }}#{{ c.type | downcase | remove: "(" | remove: ")" | remove: "@" | remove: ":" | remove: "," | replace: " ", "-" | remove: "&"  }}-calls) | {{ date }} | [**{{ c.title }}**](/{{ cohort }}/schedule#week-{{ w[0] }}) | {% if c.agenda %}{{ c.agenda }}{% endif %} |
 {%- assign w-desc = "" %}
+{%- endfor %} 
+{%- endfor %} 
+
+<script type="application/ld+json" >
```

Comment:
```suggestion
<script type="application/ld+json" >
{
  "@context": "https://schema.org",
  "@id": "{{ site.url }}/{{ cohort }}/schedule",
  "@type": "Course",
  "dct:conformsTo": "https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08",
  "description":"{{site.description}}" ,
    "hasCourseInstance": [
        {% for w in schedule.weeks %}
        {%- for c in w[1].calls -%}
        {%- if c.type != "Mentor-Mentee" -%}
       
    {
      "@context": "https://schema.org",
      "@type": "CourseInstance",
      "dct:conformsTo": "https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06",
      "courseMode": ["online", "synchronous"],
      "startDate" : "{{c.date}}",
      "endDate" :"{{c.date}}",
      "duration": "{{c.duration}}",
      "name" : "{{c.title}}"
    },

        {% endif %}
        {%- endfor -%}
        {%- endfor -%}
  ],

  "keywords": "OLS, Working Open, Cohorts",
  "name": "Cohorts Call for {{cohort}}"
}
</script >
```

Reply:
This fixed it locally.

![schema](https://user-images.githubusercontent.com/105166953/226106590-cdfafde4-315d-40ab-8342-ddba4d41ed47.png)
![Screenshot (450)](https://user-images.githubusercontent.com/105166953/226106599-9b8d9a3e-17f2-40ad-9dd2-a79808fef52f.png)

Line 50 inserts a comma between each instance.
This format 👉  `{%- -%}` 👈 ensures whitespace is not printed by the tags.